website/content/en/status/report-2025-10-2025-12/libvirt.adoc

```diff
@@ -0,0 +1,28 @@
+=== Improve libvirt support for bhyve hypervisor
+
+Links: +
+link:https://libvirt.org/drvbhyve.html[libvirt: Bhyve driver] URL: link:https://libvirt.org/drvbhyve.html[] +
+
+Contact: Roman Bogorodskiy <novel@FreeBSD.org>
+
+==== Completed work
+
+* libvirt/bhyve driver:
+** NVMe device support added.
+** PCI passthrough support added.
+** SLIRP networking support added.
+** More configuration options added: SATA nmrr/RPM, NVMe queues, VNC wait.
+* Enabled libvirt-tck https://libvirt.org/hooks.html[hooks] tests.
```

Comment:
Please use the `link` prefix (asciidoc might render the link correctly even without the prefix, but we prefer to have it explicitly)

```suggestion
* Enabled libvirt-tck link:https://libvirt.org/hooks.html[hooks] tests.
```